docs/getting-started/why-ada.md

```diff
@@ -1,40 +1,46 @@
----
```

Comment:
What's the reason behind this change?

Reply:
I think it needed a bit more content because most folks are too clueless. Also, its better to have a nicely formatted table than a long list.

Reply:
I mean this particle change from --- to ```yaml

Reply:
Hmmm. I didn't mean to change it at all.